common/app/rendering/core/Renderer.scala

```diff
@@ -24,16 +22,17 @@ class Renderer(implicit actorSystem: ActorSystem, executionContext: ExecutionCon
   implicit val timeout = Timeout(timeoutValue.seconds)
 
   def render[R <: Renderable](renderable: R): Future[Html] = {
-    (actor ? Rendering(renderable))
+    val htmlF = (actor ? Rendering(renderable))
       .mapTo[Try[String]]
-      .flatMap(Future.fromTry(_))
+      .flatMap(Future.fromTry)
       .map(Html(_))
-      .recover { case t: Throwable =>
-        val errorMessage = t.getLocalizedMessage.replaceAll("\u001B\\[[0-9]*m", "") // stripping terminal colors
-        log.error(errorMessage)
-        throw new RenderingException(errorMessage)
-      }
 
+    htmlF.failed.foreach { t =>
+      val errorMessage = Option(t.getLocalizedMessage).map(_.replaceAll("\u001B\\[[0-9]*m", "")).getOrElse("RendererError") // stripping terminal colors
+      log.error(errorMessage, t)
+    }
```

Comment:
This means that the Play error message would show the error with the formatting included, which make it not very readable. Only an issue in Dev mode though.

Reply:
It should be possible to turn off terminal formatting in bash if we need to in Dev